samples/Azure.ResourceManager.Sample/Generated/Models/VirtualMachineScaleSetExtensionData.Serialization.cs

```diff
@@ -33,10 +33,10 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
                 writer.WritePropertyName("publisher");
                 writer.WriteStringValue(Publisher);
             }
-            if (Optional.IsDefined(TypePropertiesType))
+            if (Optional.IsDefined(TypeAutoGenerated))
```

Comment:
Definitely a breaking change that we need to investigate, if its coming from another package, we need to work with them on what the change was intended to do and put in a fix before we adopt the new package

Reply:
Fixed